src/components/hooks/useOrganization.js

```diff
@@ -0,0 +1,36 @@
+import {useEffect, useContext, useState} from 'react';
+
+import {AdapterContext} from './contexts';
+
+/**
+ * A set of people in Webex.
+ *
+ * @external Organization
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/OrganizationsAdapter.js#L20}
+ */
+
+/**
+ * Returns an observable that emits organization data of the given ID.
+ *
+ * @param {string} ID ID of organization to get
+ * @returns {Organization} Data of the person
+ */
+export default function useOrganization(ID) {
+  const [organization, setOrganization] = useState({});
+  const {organizationsAdapter} = useContext(AdapterContext);
+
+  useEffect(() => {
+    const onError = (error) => {
+      // eslint-disable-next-line no-console
+      console.error(error.message);
+    };
+    const subscription = organizationsAdapter.getOrg(ID).subscribe(setOrganization, onError);
```

Comment:
In order to avoid potential state issues in the future, add a callback that creates a shallow copy for the state update, like [we do in the meetings hook](https://github.com/webex/components/blob/master/src/components/hooks/useMeeting.js#L39). As stated in the comment of the meetings hook, React doesn't recognize changes to the attributes of an object state, only a new object state will trigger a state change. By creating a shallow copy before updating the state, we can assure that any emissions from the adapter will for sure trigger a state change

```suggestion
    const onOrganization = (newOrganization) => {
      setOrganization({...newOrganization});
    };
    
    const subscription = organizationsAdapter.getOrg(ID).subscribe(onOrganization, onError);
```